### ISSUE TITLE
Bug: constrast issue for `:active` state of code wrapped in anchor

### ISSUE THREAD

**Issue by TimonVS** [mdn/yari#5469]
# Summary

There's a contrast issue with the `<code>` element contained within an `<a>` element, when the anchor is in `:active` state.

## Where did you spot the bug?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array

## What is the problem?

The contrast is insufficient, and not consistent with normal text links.

## What did you expect to happen?

`<code>` should turn white, just like normal text when the anchor is in `:active` state.

## Steps to reproduce the issue

1. Open https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array (or any other docs page containing links with code in them) in any desktop browser
2. Click on `Uint8Array()` (below "Constructor" heading)
3. Don't release the click, so that the link stays `:active`

### Device

- [x] Desktop
- [ ] Mobile: ...
- [ ] Tablet: ...
- [ ] Other: ...

### Browser

- [x] Chrome: 99.0.4844.51
- [x] Firefox: 97.0.1
- [ ] Safari: (what version?)
- [ ] Other: (what browser / version?)

### Operating system

- [ ] Windows: (what version?)
- [x] Mac OS: 12.2.1
- [ ] Linux: (what distribution / version?)

## Screenshots

https://user-images.githubusercontent.com/876666/156772153-9ce95e58-64c1-46b9-9005-3829284dfe19.mp4

![CleanShot 2022-03-04 at 14 28 04@2x](https://user-images.githubusercontent.com/876666/156771798-cdabf3fc-c143-4bd4-90e6-88a053be5b60.png)

**Comment by caugner** [comment]
This has been resolved in the mean-time:

https://user-images.githubusercontent.com/495429/201912275-ecb08994-7098-4d2c-80d1-511ff4ee09bc.mov
